src/main/java/de/tum/in/www1/artemis/domain/hestia/CodeHint.java

```diff
@@ -0,0 +1,56 @@
+package de.tum.in.www1.artemis.domain.hestia;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * A CodeHint.
+ */
+@Entity
+@DiscriminatorValue("C")
```

Comment:
Everywhere else in Artemis, the DiscriminatorValue is also always between 1 and 4 digits. So we did not want to stray from that format.
Also since this would break our current migration I don't think it would be sensible to change this, as it has already been applied to at least TS2 and TS3. 
So I think this would make more sense to change in an upcoming PR that changes this in all places, so the code stays consistent.

Reply:
You could just add a second migration to the same PR. That wouldn't be an issue.

In general, I would argue that having a short discriminator value contradicts our policy to not use characters as variable names.

Reply:
This would be something that @krusche would have to decide, as so far he advised us to only use a single changeset for this PR.
The short discriminators are present everywhere else in Artemis. And I think consistency ist more important here.
Like I said this would be better suited to be placed into another PR that also improves every other discriminator value.